.github/workflows/upmerge.yaml

```diff
@@ -1,3 +1,18 @@
+# This workflow automates the process of upmerging changes from the current release branch to the edge branch.
+# During the course of a release, the release branch is the default branch so that PRs can be immediately
+# brought into the release without waiting for a new release. This workflow merges those changes into 
+# the edge branch so that edge can be used as the basis for the next release branch.
+#
+# It is triggered manually via the workflow_dispatch event.
+#
+# The workflow performs the following steps:
+# 1. Checks out the edge branch.
+# 2. Configures git with a user name and email.
```

Comment:
Should these be added to the secrets of the repo? Would that create a problem in the forks if we do that?

Reply:
I think this is a very good idea. How about this: at the top of all the workflows we declare which environment variables are required to run the workflow. When someone forks the repo, they would go to the forked repo (in GitHub) --> settings --> Secrets and Variables --> Actions --> Variables and add the variables needed, like username and email. (Secrets can be configured the same way.)

We would have to coordinate the creation of these secrets in the origin repo before merging a change like that.